Sources/Fuzzilli/Util/Misc.swift

```diff
@@ -37,3 +37,24 @@ func measureTime<R>(_ operation: () -> R) -> (R, Double) {
     let end = Date()
     return (r, end.timeIntervalSince(start))
 }
+
+public func databuffToProgs(_ buffer: Data) throws -> [Program]{
```

Comment:
This should probably go into Fuzzilli/Protobuf and be named something like `decodeProtobufCorpus` or so?
Also I guess the encoding function should then also be moved out of Corpus.swift into here